src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java

```diff
@@ -36,9 +36,30 @@
 import com.puppycrawl.tools.checkstyle.api.FileText;
 
 /**
- * Checks the uniqueness of property keys (left from equal sign) in the
- * properties file.
+ * <p>
+ * Checks properties files for duplicated properties.
```

Comment:
> Checks properties files for duplicated properties.

This sorta sounds weird since we are saying "Check A files for duplicated A".
I like how we said `property keys` in the original.

What about instead:
`Checks properties files for duplicated property keys.`

Reply:
done